docs/section21/Rule21-5.md

```diff
@@ -0,0 +1,67 @@
+
+# Boiler - Rule 21-5  
+
+**Rule ID:** 21-5  
+**Rule Description:** The baseline building design boiler plant shall be modeled as having a single boiler if the baseline building design plant serves a conditioned floor area of 15,000sq.ft. or less, and as having two equally sized boilers for plants serving more than 15,000sq.ft.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Pass Rule 21-1.
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  1. get_zone_conditioning_category()
+  2. get_loop_zone_list_w_area()
+
+**Applicability Check:**
+
+1. Pass Rule 21-1: `if rule-21-1 == "PASS":`
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12:
+
+  - Check if P-RMR is modeled with purchased cooling or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
+    - If P-RMR is not modeled with purchased cooling (Note: Passing Rule 21-1 means P-RMR is not modeled with purchased heating.): `if NOT purchased_chw_hhw_status_dict["PURCHASED_COOLING"]:`
+
+      - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11 or 12: `PLACEHOLDER`
+
+    - Else, P-RMR is modeled with purchased cooling: `else:`
+
+      - Check if B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11 or 12: `PLACEHOLDER`
+
```

Comment:
I forgot why we need to check this. This will only be the case if project has purchased heating, but then the applicability check will not be met and the outcome will be Not Applicable?

Reply:
If baseline does not have system type that is with HHW loop then this rule is not applicable.

Reply:
I think rows 35-43 can be merged to say
Check if B-RMR is modeled with at least one air-side system that is Type-1, 1a, 5, 7, 11 or 12, continue to rule logic: PLACEHOLDER
Else, rule is not applicable: else: rule_applicability_flag = FALSE